# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/backdrop-filter-overlap.yaml
# Commit: c20cf2fa829a
# Full Hash: c20cf2fa829a19cb6b60909d05abcda40e77dd70
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-20 09:42:47
# Regressor Bug: 1764005
# File Overlap Count: 1
# Description:
#   Bug 1764005 - Add backdrop-filter primitive support r=gfx-reviewers,lsalzman,nical
#   
#   Add support for backdrop-filter primitive in WR. Each backdrop
#   filter primitive establishes a render task sub-graph. The primitive
#   instance registers itself as a resolve source to receive the
# ==============================================================================

diff -r d5b01209cf78 -r c20cf2fa829a gfx/wr/wrench/reftests/filters/backdrop-filter-overlap.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/backdrop-filter-overlap.yaml	Wed Apr 20 01:09:09 2022 +0000
@@ -0,0 +1,15 @@
+# Verify that sibling backdrop filters correctly read back intermediate results
+---
+root:
+  items:
+    - type: stacking-context
+      backdrop-root: true
+      items:
+      - image: checkerboard(0,16,16)
+        bounds: 0 0 256 256
+      - type: backdrop-filter
+        bounds: 32 32 192 192
+        filters: [invert(1)]
+      - type: backdrop-filter
+        bounds: 64 64 128 128
+        filters: [invert(1)]